src/test/java/com/rednavis/vaadin/showcase/backend/BackendServiceTest.java

```diff
@@ -2,19 +2,25 @@
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 
+import java.util.List;
+import javax.inject.Inject;
+import org.jboss.weld.junit5.EnableWeld;
+import org.jboss.weld.junit5.WeldInitiator;
+import org.jboss.weld.junit5.WeldSetup;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
-import org.mockito.junit.jupiter.MockitoExtension;
 
-@ExtendWith(MockitoExtension.class)
-public class BackendServiceTest {
+@EnableWeld
+class BackendServiceTest {
 
-  @InjectMocks
-  private BackendService service;
+  @WeldSetup
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4839326)